src/django_oapif/views.py

```diff
@@ -83,11 +83,15 @@ def get(self, request, *args, **kwargs):
             viewset = Viewset(request=request)
 
             # Returning only viewsets that match the request's permissions
-            if all(
-                permission.has_permission(request, self)
-                for permission in viewset.get_permissions()
-            ):
+            try:
+                viewset.check_permissions(request)
                 collections.append(viewset._describe(request, base_url=f"collections/"))
+            except (
+                exceptions.NotAuthenticated,
+                exceptions.AuthenticationFailed,
+                exceptions.PermissionDenied,
+            ) as e:
+                return Response(status=e.status_code)
```

Comment:
That looks strange, shouldn't we just skip unpermitted items instead of returning a response ?